src/sentry/tasks/post_process.py

```diff
@@ -571,6 +588,7 @@ def get_event_raise_exception() -> Event:
         update_event_groups(event, group_states)
         bind_organization_context(event.project.organization)
         _capture_event_stats(event)
+        _update_escalating_metrics(event)
```

Comment:
What's this change about? It seems unrelated to changing the metrics interface?

Reply:
I was just copying the code that the product team had written in order to temporarily see how it behaves with the metrics backend. would remove it once i verify its fine.

Reply:
just to highlight this again, please revert these changes